app-libs/stf/src/trusted_call.rs

```diff
@@ -399,6 +415,82 @@ impl ExecuteCall for TrustedCallSigned {
 				})?;
 				Ok(())
 			},
+			TrustedCall::ceremonies_migrate_to_private_community(who, cid, locations) => {
```

Comment:
I have a suggestion, which prevents making this file even more huge, #36. We might consider doing this as a follow-up task.

Reply:
Yes, that makes sense